.gitlab/benchmark.yml

```diff
@@ -47,7 +43,7 @@ build-adp-baseline-image:
     # Actually build.
     - aws ecr get-login-password --region us-west-2 --profile ${AWS_NAMED_PROFILE} | docker login --username AWS --password-stdin ${SMP_ECR_HOST}
     - docker buildx build
-      --platform linux/amd64,linux/arm64
+      --platform linux/amd64
```

Comment:
Changing this because SMP also doesn't run on ARM64, so there's no point in building for it.